<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateEmployee</class>
 <widget class="QWidget" name="UpdateEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Express Delivery</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#UpdateEmployee {
    background-color: #f5f5f5;
}

QPushButton#UpdateButton {
    font-size: 14px;
    background-color: transparent;
    color: #2c3e50;
    border: 1px solid #2c3e50; 
    border-radius: 8px;
    cursor: pointer;
    padding: 8px 16px;
    transition: background-color 0.3s;
}

QPushButton#UpdateButton:hover {
    background-color: #2980b9;
}

QLabel {
    font-size: 10px;
    color:#2c3e50;
}

QLineEdit {
    background-color: transparent;
    color: #2c3e50;
    border: 1px solid #2c3e50; 
    border-radius: 8px;
}

</string>
  </property>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="PerformanceLabel">
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 10px;
}</string>
    </property>
    <property name="text">
     <string>Performance:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PerformanceLineEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="ContactLabel">
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 10px;
}</string>
    </property>
    <property name="text">
     <string>Contact:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ContactLineEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="NameLabel">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Nom:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameLineEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="RoleLabel">
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 10px;
}</string>
    </property>
    <property name="text">
     <string>Role:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RoleLineEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>211</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="Hours_Label">
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 10px;
}</string>
    </property>
    <property name="text">
     <string>Volume Horaire:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HoursLineEdit"/>
  </widget>
  <widget class="QPushButton" name="UpdateButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Mettre Ã  jour</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>41</y>
     <width>52</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="IDLabel">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size: 10px;
}</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IDLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
